priv/repo/migrations/20170607142123_create_setting.exs

```diff
@@ -0,0 +1,17 @@
+defmodule CoursePlanner.Repo.Migrations.CreateSetting do
+  use Ecto.Migration
+
+  def change do
+    create table(:settings) do
```

Comment:
hmm, I see and I should say it's a very good idea. Although it makes more job rather for frontend :P
But I generally agree. @rhnonose what is your idea ?

PS : do I get this correctly 

1. Keys should be unique in the database and application level
2. in the seed we will add all the setting we need already